parser/raylib_parser.c

```diff
@@ -124,7 +124,7 @@ typedef struct DefineInfo {
 } DefineInfo;
 
 // Output format for parsed data
-typedef enum { DEFAULT = 0, JSON, XML, LUA } OutputFormat;
+typedef enum { DEFAULT = 0, JSON, XML, LUA, ALL } OutputFormat;
```

Comment:
Why adding `ALL`? Usually users will only require one output format. Please, could you remove the `ALL` option?